crates/prql-compiler/src/semantic/eval/mod.rs

```diff
@@ -277,6 +277,11 @@ impl Evaluator {
                 ExprKind::Array(array)
             }
 
+            "std.prql_version" => {
+                let ver = COMPILER_VERSION.to_string();
+                ExprKind::Literal(String(ver))
+            }
```

Comment:
Eval is a separate branch of the compiler, which is used only for `prqlc debug eval` command. It is used (probably only by me) for quick experimentation with language design and is not part of the general compilation process.

So this chunk is not needed, really.